.eslintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "globals": {
+        // Put things like jQuery, etc
+        "jQuery": true,
+        "$": true
+    },
+    "env": {
+        // I write for browser
+        "browser": true,
+        // in CommonJS
+        "node": true
+    },
+
+    "parserOptions": {
+        "ecmaVersion": 6
+    },
+    // To give you an idea how to override rule options:
+    "rules": {
```

Comment:
few other rules you might want to consider (there are lots of rules around spacing that you might want to consider):
* [comma-spacing](http://eslint.org/docs/rules/comma-spacing) - for consistency
* [space-before-function-paren](http://eslint.org/docs/rules/space-before-function-paren) - for consistency
* [space-after-function-name](http://eslint.org/docs/rules/space-after-function-name) - for consistency
* [prefer-arrow-callback](http://eslint.org/docs/rules/prefer-arrow-callback) - to avoid a mix of es6 & old js styles
* [no-console](http://eslint.org/docs/rules/no-console)  - use log instead
* [no-lonely-if](http://eslint.org/docs/rules/no-lonely-if) - cleaner code